storage/rocksdb/rdb_sst_info.cc

```diff
@@ -98,6 +98,8 @@ Rdb_sst_file_ordered::Rdb_sst_file::put(const rocksdb::Slice &key,
   DBUG_ASSERT(m_sst_file_writer != nullptr);
 
   // Add the specified key/value to the sst file writer
+#pragma GCC diagnostic push
```

Comment:
yeah, and as I understand this is supposed to be temporary. I'll look ahead in merge history to see if there is another change coming, if not, I'll add a pop.